app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
+        
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons: list[Person] = []
+    for person in people:
+        persons.append(Person(person["name"], person["age"]))
+    for person in people:
+        if "wife" in person and person["wife"] is not None:
+            Person.people[person["name"]].wife = Person.people[person["wife"]]
+            Person.people[
+                person["wife"]].husband = Person.people[person["name"]]
```

Comment:
The logic for setting `wife` and `husband` attributes assumes that the referenced person (e.g., `person['wife']`) exists in the `people` dictionary. If a referenced person is not in the input list, this will raise a `KeyError`. Consider adding a check to ensure that the referenced person exists before attempting to set these attributes.